From 758eb230361bc15ac8a335155e9b863bdb900b6e Mon Sep 17 00:00:00 2001
From: blazerpaul15 <blazerpaul15@gmail.com>
Date: Wed, 15 Jul 2026 22:29:02 +0900
Subject: [PATCH] common: Allow apps to find linearmotor service

Mark linearmotor_vibrator_service as app_api_service so Oplus Camera can
find the service for haptic feedback.

Signed-off-by: blazerpaul15 <blazerpaul15@gmail.com>

diff --git a/common/private/service.te b/common/private/service.te
index 1aba73675794a4686a6d68f8b68da7edf63dd085..2fd6c79d4dd39194a234cb8f486bdcd1338ee4e3 100644
--- a/common/private/service.te
+++ b/common/private/service.te
@@ -9,4 +9,4 @@ type ax_sandbox_service, system_api_service, system_server_service, service_mana
 type game_space_service, system_api_service, system_server_service, service_manager_type;
 type freeform_service, system_api_service, system_server_service, service_manager_type;
 type theme_engine_service, system_api_service, system_server_service, service_manager_type;
-type linearmotor_vibrator_service, system_api_service, system_server_service, service_manager_type;
+type linearmotor_vibrator_service, app_api_service, system_api_service, system_server_service, service_manager_type;
